x-pack/plugins/index_management/__jest__/client_integration/helpers/test_subjects.ts

```diff
@@ -25,6 +25,7 @@ export type TestSubjects =
   | 'ilmPolicyLink'
   | 'includeStatsSwitch'
   | 'includeManagedSwitch'
+  | 'indexActionsContextMenuButton'
```

Comment:
We can remove this key as we're not using it anywhere AFAICT

Reply:
Done